addons/main/script_macros_common.hpp

```diff
@@ -60,7 +60,7 @@
 #endif
 
 #ifndef VERSION_CONFIG
-    #define VERSION_CONFIG version = VERSION; versionStr = QUOTE(VERSION); versionAr[] = {VERSION_AR}
+    #define VERSION_CONFIG version = QUOTE(VERSION); versionStr = QUOTE(VERSION); versionAr[] = {VERSION_AR}
```

Comment:
If `versionStr` is `QUOTE(VERSION)`, then why should `version` be `QUOTE(VERSION)` as well?

Isn't it more likely that version is supposed to be a number, which works as long as there is just one comma inside the version number? After this we convert stuff like 1.00 to "1.00", even though that versionStr was already "1.00".

I don't think the choice of tool should dictate this.

I know it is a shitty way of writing this, but this we inherited from CBA_A2.

Reply:
Even CBA doesn't have a version number like that, but one with 2 dot separators, which means Arma quotes it anyways (thus is wrong).

Tool is not dictating this, Arma and the macro use in CBA and most mods is. Any proper tool will give you a warning for missing quoting. I thought stuff from A2 CBA is supposed to be rewritten anyways. ;)

I don't have a solution to provide backwards compatibility, but CBA should be without inconsistencies like this.